website/src/pages/en/index.json

```diff
@@ -75,19 +75,19 @@
     },
     "tableLegend": {
       "subgraphs": {
-        "basic": "Subgraph Studio (No issuance)",
+        "basic": "Hosted (No issuance)",
```

Comment:
i dont fully understand this change, is it because a subgraph can be hosted on multiple places?